src/doc/ESMF_quickstart.tex

```diff
@@ -461,3 +461,46 @@ \subsubsection{Building and using bundled ESMF Command Line Tools}
 instructions of the individual tools are available in the "Command Line Tools"
 section of the {\it ESMF Reference Manual}.
 
+\subsection{Building ESMF with Spack}
+
+In addition to the manual installation, ESMF can be installed through use of \htmladdnormallink{{\it Spack}}{https://spack.io} package manager. Since there are several ways to install Spack packages, this section aims to demostrate creating new isolated Spack environment to install ESMF library.
+
+The detailed documentation about Spack package manager can be found in \htmladdnormallink{{\it Spack documentation page}}{https://spack.readthedocs.io/en/latest/} and \htmladdnormallink{{\it ESMF Spack package}}{https://packages.spack.io/package.html?name=esmf} page includes ESMF package specific information.
+
+\subsubsection{Creating New Spack environment}
+
+The following set of commands can be used to clone Spack package manager, creating a new environment in the current directory and activating the environment for ESMF installation.
+
+\begin{verbatim}
+git clone -c feature.manyFiles=true --depth=2 https://github.com/spack/spack.git
+. spack/share/spack/setup-env.sh
+spack env create -d $PWD/envs/myesmf
+spack env activate $PWD/envs/myesmf
+\end{verbatim}
+
+\subsubsection{Finding Available Compilers and External Packages}
+
+Once a new Spack envronment is created and activated, available compilers and external packages can be added to the newly created environment using following Spack commands.
+
+\begin{verbatim}
+spack compiler find
+spack external find
+\end{verbatim}
+
+This will include available compilers and external packages to environment specific {\tt spack.yaml} file. In general, these are useful commands for detecting a small set of commonly-used packages but for now this is generally limited to finding build-only dependencies. The environment specific {\tt spack.yaml} file can be manually editted to include missing compilers and external packages to the environment. More information can be found in the \htmladdnormallink{{\it Spack documentation}}{https://spack.readthedocs.io/en/latest/packages\_yaml.html}.
```

Comment:
```suggestion
This will include locally available compilers and external packages to the environment specific {\tt spack.yaml} file. In general, these are useful commands for detecting a small set of commonly-used packages but for now this is generally limited to finding build-only dependencies. The environment specific {\tt spack.yaml} file can also be edited manually to include missing compilers and external packages to the environment. More information can be found in the \htmladdnormallink{{\it Spack documentation}}{https://spack.readthedocs.io/en/latest/packages\_yaml.html}.
```

Reply:
done